v1/english/main.tex

```diff
@@ -0,0 +1,845 @@
+
+Since the market resolves at the end of round $n$ with no\linebreak further stake added to any outcome, the above equation\linebreak shows the final amount of stake on the market's final outcome, $\omega_{\text{Final}}$, and the sum of all stake on all of the market's other outcomes, $\bar{\omega}_{\text{Final}}$. Note that there is ex-\linebreak actly twice as much stake on the market's final outcome as there is on all other outcomes combined.
+
+Augur redistributes all stake on the non-final out-\linebreak comes to users who staked on $\omega_{\text{Final}}$, in proportion to the amount of REP they staked. Therefore the users\linebreak who successfully filled a dispute bond in favor of $\omega_{\text{Final}}$\linebreak get a 50\% ROI on their staked REP.
+\end{proof}
+
+Next, consider the maximum number of dispute rounds required to resolve a market. Eq.~(1) is minimized when $\omega$ is chosen to be the non-tentative outcome that begins the\linebreak dispute round with the greatest amount of stake. Lemma\pagebreak~2 implies that the non-tentative outcome with the greatest amount of stake is the previous dispute round's ten-\linebreak tative outcome. Therefore, the smallest possible dispute\linebreak bond size that can be successfully filled during dispute\linebreak round $n$, where $n \geq 2$, is $B(\hat{\omega}_{n-1}, n)$.
+
+In other words, the dispute bond size grows \textit{slowest} when the same two outcomes are repeatedly disputed in favor of one another. It follows that the number of dis-\linebreak pute rounds required for a market to initiate a fork is maximized when the same two outcomes are repeatedly disputed in favor of one another. Therefore we can deter-\linebreak mine the maximum number of dispute rounds that any market may undergo before initiating a fork by finding\linebreak the maximum number of dispute rounds that can occur\linebreak in the particular case where the same two market out-\linebreak comes are repeatedly disputed in favor of one an-\linebreak other. We examine that case now.
+
+Suppose that every successful dispute bond is filled in favor of the previous dispute round's tentative outcome. Then the two tentative outcomes that are iteratively disputed in favor of one another are $\hat{\omega}_1$ and $\hat{\omega}_2$.
+
+\begin{observation}
+In the case where the same two ten-\linebreak tative outcomes are repeatedly disputed in favor of one another, $\hat{\omega}_n = \hat{\omega}_{n-2}$ for all $n \geq 3$.
+\end{observation}
+
+\begin{definition}
+Let $d$ denote the amount of stake placed\linebreak on $\hat{\omega}_1$ during the initial report. Because the tentative\linebreak outcome for each round is known in this situation, we can\linebreak simplify our notation for the dispute bond sizes. Define a\linebreak shorthand $B_n$ to denote the bond size required for round $n$, so that $B_1 \equiv 2d$ and $B_n \equiv B(\hat{\omega}_{n-1}, n)$ for all $n \geq 2$. This will make for easier reading and comprehension.
```

Comment:
The original paper used `=` here, but this uses `≡`.  Was this an intentional change or accidental?
```suggestion
Let $d$ denote the amount of stake placed\linebreak on $\hat{\omega}_1$ during the initial report. Because the tentative\linebreak outcome for each round is known in this situation, we can\linebreak simplify our notation for the dispute bond sizes. Define a\linebreak shorthand $B_n$ to denote the bond size required for round $n$, so that $B_1 = 2d$ and $B_n = B(\hat{\omega}_{n-1}, n)$ for all $n \geq 2$. This will make for easier reading and comprehension.
```